### DISCUSSION TITLE
Thanks for the best map editor

### DISCUSSION THREAD

**Discussion by Danil-Dm** [Garux/netradiant-custom#141] [General]
Hello everyone, this is the best editor of all the editors for Q3, this editor is the best that exists for Q3, thanks to the developers of this editor for such a convenient, beautiful editor, I'm developing the game Arena Unlimited this I would say is the base game and it has a very good the addon system and added many different auxiliary entities for mapping Arena Unlimited to the game, and with this editor the process of creating maps goes to a new level, I even launched the game on the compiled map through the build menu and everything works fine, how can I add support for my game to the netradiant-custom master branch? I wish the developer of this editor good luck in improving this editor. Is it possible to create maps for Xash3D in this editor? I know that you can specify a compiler for Xash3D and also add Entity types from HL and different ones for Xash3D, but what about loading textures and the BSP format? It looks like Xash3D is compatible with Valve220.

**Comment by Garux** [comment]
Thank you for kind words.
> how can I add support for my game to the netradiant-custom master branch? 

Send it and i'll include. There is no public repository with gamepacks, idk if any of these texts may have copyright issues, so processing this way.
  
  
> Is it possible to create maps for Xash3D in this editor?

Must be possible, as it is for HL, though it may be not 1st class experience, fgd and wad handling is not as smooth, as in GS centric editors. It depends on what Xash uses.

>  I know that you can specify a compiler for Xash3D and also add Entity types from HL and different ones for Xash3D, but what about loading textures and the BSP format?

Entity type in HL gamepack just says to use fgd. Textures: which format and path type Xash uses? BSP: editor doesn't work with BSP, it processes non compiled source.

> It looks like Xash3D is compatible with Valve220.

Valve220 map format is supported in editor. Other map formats may be used, if bsp compiler supports them.

**Comment by Danil-Dm** [comment]
I prepared a gamepack for Arena Unlimited, everything I did there is fully supported by the game and I removed vertex lighting from there since it is not used in Arena Unlimited 
[gamepacks.zip](https://github.com/Garux/netradiant-custom/files/11622362/gamepacks.zip)
